src/views/dog/RecapQuiz.vue

```diff
@@ -0,0 +1,216 @@
+    </div>
+
+    <div class="next-button-container">
+        <button v-if="isAnswerSelected" @click="nextQuestion" class="next-button">
+          <img src="../../assets/icon_arrow.png">
+        </button>
+    </div>
+
+    <FeedbackQuizToast :show="showToast" :message="toastMessage" />
+
+    <div class="modal">
+      <div v-if="showModal" class="modal-overlay">
+        <div class="modal-content">
+          <h2>Toll gemacht! Du hast alle Fragen richtig beantwortet. Du kennst Rocky wirklich gut!</h2>
+          <button @click="finishQuiz" class="finish-button">Beenden</button>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, computed } from 'vue';
+import QuestionComponent from '@/components/QuizQuestion.vue';
+import AnswerComponent from '@/components/QuizAnswer.vue';
+import FeedbackQuizToast from '@/components/FeedbackQuizToast.vue';
+import quizData from '@/config/quizConfig';
+import { useRouter } from 'vue-router';
+
+const currentQuestionIndex = ref(0);
+const selectedAnswer = ref<string | null>(null);
+const isAnswerSelected = ref(false);
+
+const isCorrect = ref<boolean[]>(new Array(quizData[currentQuestionIndex.value].answers.length).fill(false));
+const isIncorrect = ref<boolean[]>(new Array(quizData[currentQuestionIndex.value].answers.length).fill(false));
+
+const showToast = ref(false);
+const toastMessage = ref('');
+
+const router = useRouter();
+const showModal = ref(false);
+
+const currentQuestion = computed(() => {
+  return quizData[currentQuestionIndex.value];
+});
+
+const handleAnswerSelected = (selectedAnswerText: string, index: number) => {
+  showToast.value = true;
+  selectedAnswer.value = selectedAnswerText;
+
+  if(selectedAnswer.value === currentQuestion.value.correctAnswer && selectedAnswer.value === selectedAnswerText) {
+    isCorrect.value[index] = true;
+    isIncorrect.value.fill(true);
+    isIncorrect.value[index] = false;
+    
+    isAnswerSelected.value = true;
+    toastMessage.value = 'Gut gemacht! Drücke auf den Pfeil.';
+  } else {
+    isIncorrect.value[index] = true;
+    toastMessage.value = 'Schade, das ist leider nicht richtig. Versuch es doch noch einmal.';
+  }
+  setTimeout(() => {
+    showToast.value = false;
+  }, 3000);
+};
+
+const nextQuestion = () => {
+  isAnswerSelected.value = false;
+  selectedAnswer.value = null;
+  isCorrect.value.fill(false);
+  isIncorrect.value.fill(false);
+
+  if (currentQuestionIndex.value < quizData.length - 1) {
+    currentQuestionIndex.value++;
+  } else {
+    showModal.value = true;
+  }
+};
+
+const finishQuiz = () => {
+  router.push(`/pets/dog/stages/1`);
+};
+
+const progress = computed(() => {
+  const progressArray = [];
+  for (let i = 0; i < quizData.length; i++) {
+    if (i <= currentQuestionIndex.value) {
+      progressArray[i] = true;
+    } else {
+      progressArray[i] = false;
+    }
+  }
+  return progressArray;
+});
+
+</script>
+
+<style scoped>
+.quiz-container {
```

Comment:
Please remove the background color.

Is this container needed at all? Could all the logic maybe just go into the question-section class?